### PR TITLE
Create spicyaxolotl.json

### PR DESCRIPTION
**Pull request by Spicy-Axolotl**

### CODE REVIEW

**Review by AbubakarKang — DISMISSED** [review]
You have to write at least one contact detail.

File: domains/spicyaxolotl.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "My personal website",
+  "repo": "https://github.com/Spicy-Axolotl/spicy-axolotl.github.io",
+  "owner": {
+    "username": "Spicy-Axolotl",
+    "email": "",
```
Comment (MaskDuck) [inline domains/spicyaxolotl.json:6]: hold on, can i get some contact details such as your discord/your email/etc beside of your github pls?

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/spicyaxolotl.json (hunk 1 above)
Comment (Spicy-Axolotl) [reply domains/spicyaxolotl.json:6]: Here is my discord: Axolotl (mycoolface256)#4736

Comment (MaskDuck) [reply domains/spicyaxolotl.json:6]: no, write them into the json file.

Comment (Spicy-Axolotl) [reply domains/spicyaxolotl.json:6]: There you go.

Comment (MaskDuck) [reply domains/spicyaxolotl.json:6]: also feel free to join our discord if you are interested

**Review by MaskDuck — DISMISSED** [review]

**Comment by Spicy-Axolotl** [comment]
Alright thank you!

**Review by MaskDuck — APPROVED** [review]

File: domains/spicyaxolotl.json (hunk 1 above)
Comment (Spicy-Axolotl) [reply domains/spicyaxolotl.json:6]: Ok.

**Pull request merged**